doc/developer-guide/plugins/remap-plugins.en.rst

```diff
@@ -80,12 +80,46 @@ multiple times for the rule, once for each invocation. Only the value store in :
 available when the rule is actually matched. In particular the plugin arguments will not be
 available.
 
-Calls to :func:`TSRemapNewInstance` are serialized.
+Calls to :func:`TSRemapNewInstance` are serialized. All calls to :func:`TSRemapNewInstance`
```

Comment:
Two types of serialization here - one is that calls to `TSRemapNewInstance` are serialized of themselves, that is it will never be the case that there are two calls on two threads simultaneously. That was the meaning of the original comment. There's also the text you put here, which is about _sequencing_. I think it needs to be clear that both of these are guarantees.

Reply:
Actually the statement was clear from your original version (about multiple threads calling simultaneously). I did not think I would spoil that by following that with a statement about "sequencing", that the series will be all done before a non-initialization per-request call to `TSRemapDoRemap`. Added "guaranteed" statements in PR #5790 since unintentionally missed adding it with this PR. HTH